nautobot/dcim/api/views.py

```diff
@@ -622,22 +623,23 @@ class InventoryItemViewSet(NautobotModelViewSet):
 
 #
 # Connections
+# TODO: remove these in favor of using the ConsolePort/PowerPort/Interface API endpoints and/or Cable endpoint.
 #
 
 
-class ConsoleConnectionViewSet(ListModelMixin, GenericViewSet):
+class ConsoleConnectionViewSet(ModelViewSetMixin, ListModelMixin, GenericViewSet):
```

Comment:
I *think* you can probably get rid of `GenericViewSet` since you're adding `ModelViewSetMixin`.

Reply:
Thanks for asking - I checked and our `ModelViewSetMixin` is a pure mixin class, no parent class. I added it to these viewsets because it's what provides the custom `list` override to render to CSV; if we can get rid of that override (as mentioned above) then these can get rid of the mixin again.